icon_validator/rules/plugin_validators/help_input_output_validator.py

```diff
@@ -4,6 +4,92 @@
 from icon_validator.rules.validator import KomandPluginValidator
 from icon_validator.exceptions import ValidationException
 
+from datetime import datetime
+
+
+def detect_valid_datetime(list_entry: str) -> bool:
+    """
+    This function determines if a valid datetime following
+    iso format is detected in a string
+
+    :param list_entry: The entry of the list to be checked for datetime validity
+    :type list_entry: str
+
+    :rtype: bool
+    """
+    try:
+        datetime.fromisoformat(list_entry)
+        return True
+    except ValueError:
+        return False
+
+
+def convert_to_valid_datetime(table_string: str) -> str:
```

Comment:
please test that method separately, test what happens when you input different formats